tests/shared/src/test/scala/cats/tests/OptionTSuite.scala

```diff
@@ -378,6 +378,21 @@ class OptionTSuite extends CatsSuite {
     }
   }
 
+  test("OptionT.whenM[Id, A] consistent with Option.when") {
```

Comment:
can we add a similar test for `filterF[Id, A] consistent with Option.filter`?

Reply:
Done!